content/documentation/pages/5-feature-guides/2-batch/3-partitioning.md

```diff
@@ -1,7 +1,260 @@
 ---
 path: 'feature-guides/batch/partitioning/'
-title: 'Data Partitioning'
-description: 'How to use partitioning'
+title: 'Remote Batch Partitioning'
+description: 'How to use remote batch partitioning'
 ---
 
-Introduction to partitioning
+# Introduction to remote partitioning
+
+The Batch Developer Guide showed you how to create the typical single threaded Spring Batch application.
+While this is a great solution for most batch applications, there are cases where a particular step may take a significant amount time to do the work required.
+Spring Batch has a solution that allows a [Batch Job](https://docs.spring.io/spring-batch/4.0.x/reference/html/domain.html#job) to partition a [Step](https://docs.spring.io/spring-batch/4.0.x/reference/html/domain.html#step) execution, where each partition handles a segment of work.
```

Comment:
I remember asking about it elsewhere. Sorry for the same question. Should we not use `current` instead of `4.0.x`? Maybe also parameterize this in the versioning.json or something to be safe?

Reply:
resolved